doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -347,14 +347,27 @@ Other libraries
 
 * :ref:`event_manager`:
 
-  * To add :c:func:`event_manager_event_size` function with corresponding :kconfig:option:`CONFIG_EVENT_MANAGER_PROVIDE_EVENT_SIZE` option.
+  * Added:
+
+    * Event type flags to represent if event type should be logged, traced and has dynamic data.
+       To update your application, pass a flag variable as a parameter in :c:macro:`EVENT_TYPE_DEFINE` instead of ``init_log``:
+       You can use :c:macro:`EVENT_FLAGS_CREATE` to set multiple flags:
```

Comment:
```suggestion
    * Event type flags to represent if event type should be logged, traced and has dynamic data.
      To update your application, pass a flag variable as a parameter in :c:macro:`EVENT_TYPE_DEFINE` instead of ``init_log``.
      You can use :c:macro:`EVENT_FLAGS_CREATE` to set multiple flags:
```